web/src/composebox_typeahead.js

```diff
@@ -452,19 +452,19 @@ export const dev_only_slash_commands = [
 
 export const slash_commands = [
     {
-        text: $t({defaultMessage: "/me is excited (Display action text)"}),
+        text: $t({defaultMessage: "/me (Action message)"}),
         name: "me",
         aliases: "",
-        placeholder: $t({defaultMessage: "is …"}),
+        placeholder: $t({defaultMessage: "…"}),
```

Comment:
What is the reason for this change? I don't see anything about it in the issue.

Reply:
My bad, I should not have changed the placeholder. I will fix it. Is there anything that needs a fix?

Reply:
Changed placeholder back to its original value.